docs/rfcs/1005-realm-key-rotation.md

```diff
@@ -0,0 +1,821 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+# No-reencryption Realm key rotation
+
+## 0 Abstract
+
+This RFC proposes a way to rotate realm keys without requiring re-encryption of all data.
+
+## 1 Background
+
+The current realm key rotation system works by replacing the realm key, then re-encrypting
+all realm's vlobs with the new key.
+
+The rational for this approach was that the client only have to deal with a single key at any given time.
+However this has some downsides:
+
+- Reencrypting all vlobs takes some time (though not that much in practice given it's only
+  metadata, the empirical rule being 1000 times less metadata than actual data in Parsec).
+- This concept must be exposed to the user so that he can decide when to do the reencryption
+  maintenance procedure. This is a concept complex to explain so not great for UX :/
+- A realm encryption revision index is kept server-side and send to the client to ensure
+  it is not using the wrong key to decrypt
```

Comment:
Not sure I understand this. Is this index related to the fact that, because of maintenance, there may be two keys? 
If so, then it is already explained below (the last paragraph of section 1) and I would simply remove the whole item from the list here.

```suggestion
```